src/components/Image/tests/Image.test.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from 'react';
+import Image from '..';
+import { mount } from 'enzyme';
+
+describe('source', () => {
+    it('should verify the source type of the string', () => {
+        const imageWrapper = mount(<Image source="string" title ="" alt=""/>);
+        expect(typeof imageWrapper.prop('source')).toBe('string');
+    });
+
+    it('should verify the source of the Image url', () => {
+        const imageWrapper = mount(<Image source="https://www.w3schools.com/css/trolltunga.jpg" title ="" alt=""/>);
+        expect(imageWrapper.prop('source')).toBe('https://www.w3schools.com/css/trolltunga.jpg');
+    });
+});
+
+describe('alt', () => {
```

Comment:
For each property, add following cases.

1) Not set.
2) Set to valid/true case
3) Set to undefined/false case. Refer Chip Component for cases.